imports/ui/views/users/UsersList/index.js

```diff
@@ -0,0 +1,2 @@
+import ULSC from './UsersListStateContainer'
```

Comment:
MALTWGC (Maybe abbreviations like this will get confusing?) 
Why not just use the full name?